_includes/shared/markup/dev/flink_sql_rowtime_aggregation.adoc

```diff
@@ -0,0 +1,2 @@
+IMPORTANT: If you have a table that does not have an explicit timestamp column you can use the `SYSTEM` column `$rowtime` for the `DESCRIPTOR` parameter when performing windowed aggregation with Flink SQL in Confluent Cloud.
```

Comment:
just a suggestion: `NOTE` rather than `IMPORTANT` bc the red background that you get for `IMPORTANT` notes feels a little loud to me